xrpl/models/transactions/xchain_account_create_commit.py

```diff
@@ -0,0 +1,29 @@
+"""Model for a XChainAccountCreateCommit transaction type."""
+
+from dataclasses import dataclass, field
+
+from xrpl.models.amounts import Amount
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import require_kwargs_on_init
+from xrpl.models.xchain_bridge import XChainBridge
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True)
+class XChainAccountCreateCommit(Transaction):
+    """Represents a XChainAccountCreateCommit transaction."""
+
+    xchain_bridge: XChainBridge = REQUIRED  # type: ignore
```

Comment:
Why do these have `# type: ignore` comments?

Reply:
It's a quirk of typing in Python. IIRC you can't have a dataclass that has required variables that inherits from a dataclass that has optional variables. So this was our solution to get around that - `REQUIRED` is a special type, but typing has to be ignored since `REQUIRED` is not of type `XChainBridge`. See https://github.com/XRPLF/xrpl-py/blob/master/xrpl/models/required.py
Changing anything about that setup is way out of the scope of this PR.